src/documentation/language/malloy4.malloynb

```diff
@@ -0,0 +1,67 @@
+>>>markdown
+# Malloy 4.0
+
+We are very excited to announce version 4.0 of the Malloy language.
+
+There was never an official "3.0", we are calling this release 4.0 because this is the fourth major
+renovation of the language. It could also be thought of as "1.0" because this is when we stop using
+"experimental" to describe the language. At this point we have experimented enough.  We believe
+Malloy is now the best language for working with data stored in an SQL server.
+
+## The 4.0 Release Timeline
+
+Before 4.0, when Malloy was in the experimental phase, it was considered acceptable to make breaking
+changes to the language as we discovered better ways to express data computations.  Beginning with
+4.0, it is our intention to keep code compatibility for existing code, even as we continue to add
+capabilities to the language.
+
+The rollout of 4.0 will happen in a way which should make it relatively easy for users to upgrade their code.
+
+* **Today:** Both 4.0 and pre-4.0 Malloy code are usable in the Pre-Release Malloy VS Code Extension
+  * If you are using a language construct which will not work in 4.0, the IDE will show you a warning.
+  * The [Documentation](..) describes the 4.0 syntax.
```

Comment:
Link checker is complaining that this link is not valid. You should point to a specific document, e.g. `[Documentation](../index.malloynb)`.